### PR TITLE
[Docs] Add 'Connecting to Remote Providers' section in Troubleshooting Guide

### LINKED ISSUE #4597: [Docs] Add "Connecting to Remote Providers" section in Troubleshooting Guide
#### Current State
The troubleshooting guide is missing guidance on Remote Providers.

#### Desired State
- New Remote Provider section
- New "Connecting to Remote Providers" sub-section with notes from recent troubleshooting session (reference - https://layer5io.slack.com/archives/CVBGPTFJN/p1636338508049800?thread_ts=1636045480.029500&cid=CVBGPTFJN)

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by gyohuangxin**
**Description**

Add 'Connecting to Remote Providers' section in Troubleshooting Guide

This PR fixes #4597

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 9e1af418690cff4c9011a10ad706352ca94d0d31 at: https://618e11c4d1675e1f4cad4d59--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 3e3b490178bff1984fe30cd81d2d83f306fbd7da at: https://618e1288742b2918843dadd5--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 744bf47e7af2594cd06fba1d3399c7ec752519dc at: https://618e132494dbd03373200f42--meshery-docs-preview.netlify.app

File: docs/pages/reference/troubleshooting/installation.md [hunk 1]
```diff
@@ -38,3 +38,26 @@ For details about the state of the Meshery Server subscription see the http moni
 #### See Also
 
 - [Error Code Reference](/reference/error-codes)
+
+## Meshery Remote Providers
+
+Once Meshery is installed, the remote provider "Meshery" can be chosen from UI or by using command `mesheryctl system login`:
```
Comment (Mr-DG-Wick) [inline docs/pages/reference/troubleshooting/installation.md:44]:
```suggestion
Once Meshery is installed, the remote provider "Meshery" can be chosen from UI or by using the command `mesheryctl system login`:
```

File: docs/pages/reference/troubleshooting/installation.md [hunk 2]
```diff
@@ -38,3 +38,26 @@ For details about the state of the Meshery Server subscription see the http moni
...
+
+![Providers](/assets/img/providers/provider_screenshot.png)
```
Comment (Mr-DG-Wick) [inline docs/pages/reference/troubleshooting/installation.md:46]: I think there seems to be an issue with the image. It isn't visible to me in the preview mode.

**Review by Mr-DG-Wick — CHANGES_REQUESTED** [review]
Thanks for taking this up! @gyohuangxin . Suggested some minor suggestions.

**Comment by leecalcote** [comment]
yay, @gyohuangxin! 🎈

File: docs/pages/reference/troubleshooting/installation.md [hunk 3]
```diff
@@ -38,3 +38,26 @@ For details about the state of the Meshery Server subscription see the http moni
...
+
+```
+➜  ~ mesheryctl system login
+Use the arrow keys to navigate: ↓ ↑ → ←
+? Select a Provider:
+  ▸ Meshery
+    None
+```
+
+If you cannot see "Meshery" provider and find such error logs in meshery pod's log, please make sure that the meshery pod has the connections to "https://meshery.layer5.io" to initialize remote provider.
```
Comment (leecalcote) [inline docs/pages/reference/troubleshooting/installation.md:56]:
```suggestion
If you cannot see "Meshery" Remote Provider and find such error logs in Meshery Server's logs (`mesheryctl system logs`), please make sure that Meshery Server is able to reach "https://meshery.layer5.io" in order to initialize the "Meshery" Remote Provider.
```

**Review by leecalcote — APPROVED** [review]
This is great. Thank you, @gyohuangxin 👍

**Comment by gyohuangxin** [comment]
@leecalcote @Mr-DG-Wick Thank you for your reviews.

**Pull request merged**